content/en/community/ways-to-contribute.md

```diff
@@ -22,61 +22,58 @@ Other ways to contribute to open source in which you don’t necessarily have to
 
 -->
 
-## 5 ways to contribute to open source without coding
+## 5 ways to contribute to Scientific Python without coding
 
 <!--
 Hello everyone, I’m Juanita.
 Welcome to the Scientific Python videos! Today I will be sharing with you some ways in which you can contribute to open source software projects without actually having to code.
 -->
 
-Open source software is code designed in a way that it is accessible to the public, everyone can view, modify, and distribute it easily, according to the terms of permissive software licenses.
+Scientific Python software is code designed in a way that it is accessible to the public, everyone can view, modify, and distribute it easily, according to the terms of permissive software licenses.
```

Comment:
Maybe also something along the lines of:

"All projects have a license that determine what you can and cannot do; for Scientific Python these licenses are typically very straightforward: you may use and modify the software, as long as you give credit to the original authors."

Reply:
See what you think about what I did.